showcase/README.md

```diff
@@ -38,18 +40,30 @@ $ solana config set -u devnet
 ```
 
 ### SOL DID Client
+The SOL DID client allows you to manage DID documents on Solana.
+
 ```shell
 $ yarn global add @identity.com/sol-did-client
 ```
 
 ### Cryptid Client
+The Cryptid client allows you to manage your Cryptid account on the cli.
+
+[See here](https://github.com/identity-com/cryptid) for more information on Cryptid.
+
 ```shell
 $ yarn global add @identity.com/cryptid-cli
 
 $ cryptid --help
 ```
 
 ### Gateway Client
+The gateway client allows managing of gateway tokens. A gateway token is required for a user to prove that they have
+been verified by a gatekeeper before being allowed to use certain on-chain applications. The rules required for a 
+gatekeeper to issue a token could include things like age checks, OFAC checks, etc.
+
+More information can be found [here](https://github.com/identity-com/on-chain-identity-gateway).
```

Comment:
this might be a better link https://identity-com.github.io/on-chain-identity-gateway